src/pages/AddRecipe.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom'
+
+import RecipeForm from '../components/RecipeForm';
+import { ButtonField } from '../styles/AddRecipe';
+import { Button } from '../styles/Recipe';
+
+const linkStyle = {
+  margin: "0.5rem",
+  textDecoration: "none",
+  color: '#333'
+};
+function AddRecipe() {
```

Comment:
IN TK we use styled component, I advise you to try them out if you don't know them already. https://styled-components.com/docs/basics#installation